pkg/util/lsof/lsof_linux.go

```diff
@@ -0,0 +1,383 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package lsof
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/prometheus/procfs"
```

Comment:
Good thing that we already had this dependency in the Agent 😉

Reply:
To be fair it has almost no dependency itself, and I don't think it needs CGO, so I think it's a pretty lightweight dependency anyway

Reply:
Yeah it's mostly a text parser for `/proc`